databases/business-sync/fragment-shoebox-item.js

```diff
@@ -62,6 +62,74 @@
           }
         }
       }
+    },
+    annotations: {
+      type: 'array',
+      required: false,
+      arrayElementsValidator: {
+        type: 'object',
+        required: true,
+        propertyValidators: {
+          // indicates what type of annotation this is
+          // Only embedded is supported to start
+          type: {
+            type: 'enum',
+            immutable: true,
+            required: true,
+            predefinedValues: [ 'embedded' ]
+          },
+          // a simple tag describing what type data this annotation is
+          // example:  'user-metadata', 'ocr-analysis', 'model-XYZ-classification'
+          dataType: {
+            type: 'enum',
```

Comment:
For some of the proposed examples, the field is really capturing two concepts.  Ie, for 'user-metadata', it's capturing both the source and type of the data.  I think this is really just an issue with choosing dataType keys carefully - the source should be captured in the modifications/who was responsible for creating/saving this annotation.  Will update `user-metadata` to simply `metadata`.